strongbox-commons/src/main/java/org/carlspring/strongbox/booters/TempDirBooter.java

```diff
@@ -75,14 +66,14 @@ private void createLockFile()
         Files.createDirectories(lockFile.getParent());
         Files.createFile(lockFile);
 
-        logger.debug(" -> Created lock file '{}'...", lockFile.toAbsolutePath());
+        logger.debug(" -> Created lock file '" + lockFile.toAbsolutePath().toString() + "'...");
```

Comment:
Please, use the built-in formatting for loggers:
`logger.debug(" -> Created lock file '{}'...", lockFile.toAbsolutePath());`